### ISSUE TITLE
[TASK] Deprecated calls, which will be removed in CMS v.9

### ISSUE THREAD

**Issue by dkd-kaehm** [TYPO3-Solr/ext-solr#1165]
# Here is the place for next releases.

## Will be removed in CMS v. 9:
 
- [x] in `solr/Classes/IndexQueue/PageIndexerRequestHandler.php`: line 19 and 108 in #1367 
  - [x] `Using the persistent object token "&" when resolving "&ApacheSolrForTypo3\Solr\IndexQueue\PageIndexerRequestHandler->shutdown"  for GeneralUtility::callUserFunc() is deprecated since TYPO3 v8. Make sure to implement SingletonInterface to achieve the same functionality. This functionality will be removed in TYPO3 v9 and will then result in a fatal PHP error.`
  - [x] `Using the persistent object token "&" when resolving "&ApacheSolrForTypo3\Solr\IndexQueue\PageIndexerRequestHandler->run"  for GeneralUtility::callUserFunc() is deprecated since TYPO3 v8. Make sure to implement SingletonInterface to achieve the same functionality. This functionality will be removed in TYPO3 v9 and will then result in a fatal PHP error.`
- [x] `solr/Resources/Private/Templates/Administration/Index.html + Resources/Private/Templates/Administration/Index.html`
  - [x] `Setting "enableClickMenu" in Container ViewHelper...` is gone somewhere
  - [x] `Setting "loadjQuery" and "jQueryNamespace" in PageRenderer` in #1446 
- [x] `many files` 
  - [x] `DatabaseConnection a.k.a. $["TYPO3_DB"] has been marked as deprecated in TYPO3 v8 and will be removed in TYPO3 v9. Please use the newly available ConnectionPool and QueryBuilder classes.`
- [x] `solr/Classes/Backend/IndexingConfigurationSelectorField.php` in #1452 
  - [x] `TYPO3\CMS\Backend\Form\FormResultCompiler->JStop() - since TYPO3 v8, will be removed in TYPO3 v9`

**Comment by timohund** [comment]
Thanks for the summary i've added the issue to milestone 7.0

**Comment by timohund** [comment]
I'll split this into several issues to make it easier to work on this in parallel.

**Comment by timohund** [comment]
All deprecations from this issue should be resolved. If you find any leftover please create a new issue.
